README.md

```diff
@@ -1,152 +1,185 @@
-    "time": "2020-06-07T07:02:15.64489Z",
-    "contenttype": "application/json",
-    "data": {
-        "State": "OPEN",
-        "ProblemID": "ab81-941c-1245",
-        "ProblemTitle": "ansibletower service test event",
-        "project": "sockshop",
-        "stage": "production",
-        "service": "carts"
+1. Deploy Ansible Tower action provider
+    
+    Install the Ansible Tower service for Keptn by applying the provided yaml file (latest version).
+    
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/keptn-sandbox/ansibletower-service/master/deploy/service.yaml
+    ```
+    This will install the Ansible Tower service in the Keptn namespace.
+
+# TODO describe how to setup a demo playbook in Ansible Tower. we could add the demo playbook also in this repo for users to reuse
+
+# Demo use case: Onboard sample application
+
+1. Clone the Keptn demo repo: 
+    ```
+    git clone https://github.com/keptn/examples.git
+    ```
+
+1. Move to the correct folder: 
+    ```
+    cd examples/onboarding-carts
+    ```
+
+1. If you are running a command-line remote to the cluster, open a new prompt and forward the port for the configuration service.
+
+    ```
+    kubectl -n keptn port-forward service/api-gateway-nginx 8080:80 >/dev/null &
+    ```
+
+1. The next steps assume you already have Keptn installed (both the CLI and in the cluster), check pre-requisites if not.
+
+    ```
+    keptn create project sockshop --shipyard=./shipyard.yaml
+
+    keptn onboard service carts-db --project=sockshop --chart=./carts-db --deployment-strategy=direct
+
+    keptn onboard service carts --project=sockshop --chart=./carts
+
+    keptn send event new-artifact --project=sockshop --service=carts-db --image=docker.io/mongo --tag=4.2.2
+
+    keptn send event new-artifact --project=sockshop --service=carts --image=docker.io/keptnexamples/carts --tag=0.11.1
+    ```
+
+1. Next we are going to write a simple remediation file that we are adding to Keptn. Please copy the following contents into a file called `remediation-tower.yaml`. Please refer to https://keptn.sh/docs/concepts/automated_operations for more information.
+
+    Please note to change the value of the `JobTemplate` to trigger to the value of your job template in Ansible Tower.
+
+    ```
+    apiVersion: spec.keptn.sh/0.1.4
+    kind: Remediation
+    metadata:
+    name: remediation-service
+    spec:
+    remediations:
+        - problemType: ansibletower service test event
+        actionsOnOpen:
+            - name: Test event
+            description: Testing the Ansible Tower integration
+            action: job_template_launch
+            value:
+                JobTemplate: "9"
+    ```
+
+1. Add remediation file to keptn.
+
+    ```
+    keptn add-resource --project=sockshop --stage=production --service=carts --resource=remediation-tower.yaml --resourceUri=remediation.yaml
+    ```
+
+    Note: the name of the remediation file must be `remediation.yaml`.
```

Comment:
@jetzlstorfer the way you rewrote using `remediation-tower.yaml` as file name was much better but now the Note here is confusing. How about we change it to: 
```suggestion
    Note: it is mandatory that the `resourceUri` paramter be exactly `remediation.yaml`
```